maro/data_lib/dump_csv_converter.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+
+import os
+import threading
+import urllib
+from datetime import datetime
+from math import floor
+from pathlib import Path
+from shutil import copyfile
+
+import numpy as np
+import pandas as pd
+import yaml
+
+
+class DumpConverter:
+    """ This class is used for convert binary snapshot dump content to CSV format. """
+
+    def __init__(self, parent_path="", scenario_name="", prefix="epoch_", serial=0):
+        super().__init__()
+        self._parent_path = parent_path
+        self._prefix = prefix
+        self._serial = serial
+        self._scenario_name = scenario_name
+        self._manifest_created = False
+        self._mapping_created = False
+
+    def __generate_new_folder(self, parent_path):
```

Comment:
correct me if I'm wrong, I thought we should avoid using double underscores to specify an object in development